features/importers/myhelsinki_places/importer.py

```diff
@@ -109,6 +115,27 @@ def import_feature_tags(self, feature: Feature, tags: Iterable[dict]):
         feature_tags = [tag for tag in map(self.tag_mapper.get_tag, tags) if tag]
         feature.tags.set(feature_tags)
 
+    def import_feature_contact_info(self, feature: Feature, address: dict):
+        """Imports contact info for the given feature."""
+
+        if not (
+            address["street_address"]
+            or address["postal_code"]
+            or address["municipality"]
+        ):
+            # Delete existing address if source doesn't provide this information
+            if hasattr(feature, "address"):
+                feature.address.delete()
```

Comment:
I guess this should be `contact_info` instead `address`?

Reply:
I was supposed to fix it, but apparently I got distracted while refactoring the related if clause. :sweat:

Reply:
Nice spot.